### PR TITLE
Update UPGRADE.md

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 
-### !!!IMPORTANT!!!
+> This file contains the upgrade notes. These notes highlight changes that could break your
+> application when you upgrade the package from one version to another.
 
-The following upgrading instructions are cumulative. That is,
-if you want to upgrade from version A to version C and there is
-version B between A and C, you need to following the instructions
-for both A and B.
+<div class="alert alert-warning" style="background-color: #e2a03f14;display: flex;padding: 1rem 1rem;margin-bottom: 1rem;border: none;border-left: 3px solid #e2a03f;border-radius: 4px;"><span style="color:red; font-weight: bold;"> **!!! IMPORTANT !!!**<br>The following upgrading instructions are CUMULATIVES. That is, if you want to upgrade from version A to version C and there is  version B between A and C, you need to following the instructions for both A and B.</span>
```
Comment (vjik) [inline UPGRADE.md:6]: Without HTML is better.

**Review by vjik — CHANGES_REQUESTED** [review]

File: UPGRADE.md (hunk 1 above)
Comment (luizcmarin) [reply UPGRADE.md:6]: any github restrictions, difficulties in viewing in some countries or systems, any open source recommendations or just to keep it as simple as possible?
Please explain why you think so

Comment (vjik) [reply UPGRADE.md:6]: It's difficult when see it in text mode.

Comment (luizcmarin) [reply UPGRADE.md]: done

**Review by vjik — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**